### ISSUE TITLE
(mattermost-desktop) Auto-update disabled and updates are out

### ISSUE THREAD

**Issue by StingRayZA** [chocolatey-community/chocolatey-coreteampackages#1318]
mattermost-desktop was disabled by #1305 . There's now updates to the application (4.2.3).
Re-enabling auto-update will allow these to be caught.

## Expected Behavior
mattermost-desktop package 4.2.3 gets built

## Current Behavior
mattermost-desktop static at 4.2.1

## Possible Solution
re-enable auto-builds (even temporarily?)

## Steps to Reproduce (for bugs)
N/A

## Context
I'd like to update my MM client, and can't do so via chocolatey.

**Comment by AdmiringWorm** [comment]
@wget can you confirm that it is now okay to enable automatic updates again for the mattermost-desktop package?

**Comment by majkinetor** [comment]
The update is enabled but will take random time due to the #1217

**Comment by wget** [comment]
@AdmiringWorm Yes sorry for the late reply :/ Yes, the new 4.3 is still on hold at Mattermost. We detected a regression late in the development process regarding toast notifications on Windows, so we had to postpone the release of the 4.3 (which has been demoted from RC0 to beta :/ ).
We may need to disable the update process again in the weeks (I hope not months) to come though. Will let you know via the core team package chat on Gitter any way :)
